src/app/genes/gene-similar/gene-similar.component.ts

```diff
@@ -67,19 +68,16 @@ export class GeneSimilarComponent implements OnInit {
     ngOnInit() {
         this.cols = [
             { field: 'hgnc_symbol', header: 'Gene name' },
-            { field: 'brain_regions', header: 'Brain Regions' },
-            { field: 'num_brain_regions', header: 'Number of Brain Regions' },
-            { field: 'nominations', header: 'Nominated Target' },
+            { field: 'brain_regions_display_value', header: 'Brain Regions' },
+            { field: 'num_brain_regions_display_value', header: 'Number of Brain Regions' },
+            { field: 'nominated_target_display_value', header: 'Nominated Target' },
             { field: 'isIGAP', header: 'Genetic Association with LOAD'},
             { field: 'isChangedInADBrain', header: 'RNA Expression Change'},
             { field: 'haseqtl', header: 'Brain eQTL' },
-            { field: 'druggability', subfield: 'pharos_class', header: 'Pharos Class'},
-            { field: 'druggability', subfield: 'sm_druggability_bucket',
-                header: 'Small Molecule Druggability'},
-            { field: 'druggability', subfield: 'safety_bucket',
-                header: 'Safety Rating'},
-            { field: 'druggability', subfield: 'abability_bucket',
-                header: 'Antibody Modality'}
+            { field: 'pharos_class_display_value', header: 'Pharos Class' },
+            { field: 'sm_druggability_display_value', header: 'Small Molecule Druggability' },
+            { field: 'safety_rating_display_value', header: 'Safety Rating' },
+            { field: 'ab_modality_display_value', header: 'Antibody Modality' },
```

Comment:
Instead of using subfields here, I'm using the display field approach from genes-list.